packages/framework/tinylicious-client/README.md

```diff
@@ -1,4 +1,4 @@
-# @fluid-experimental/tinylicious-client
+# @fluidframework/tinylicious-client
 
 The tinylicious-client package provides a simple and powerful way to consume collaborative Fluid data with the Tinylicious service.
 
```

Comment:
"This package is marked as experimental and currently under development. The API surface is currently under going drastic braking changes with no guarantees on compatibility." can be removed